### ISSUE TITLE
Metadata status: startup message is not announced if Studio windows other than playlist viewer is focused when NVDA starts

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#83]
Hi,

Consider the following scenario: user loads Studio, moves to Options (Control+O), then restarts NVDA. If NvDA is told to announce metadata streaming status at startup, this message is not announced when NVDA restarts.

Technical: regression caused by #82: strict checking for Studio window in foreground event isn't honored when restarting NVDA while focused on windows other than playlist viewer. Thus one needs to make sure playlist viewer is in fact one of the foreground windows before proceeding with metadata streaming status announcement (or any startup message that requires careful timing). For this bug, no readme entry is required, and it'll be backported to LTS18.

Thanks.

**Comment by josephsl** [comment]
Resolved in 2019. Thanks.
